src/fixate/drivers/ftdi.py

```diff
@@ -267,12 +267,10 @@ def baud_rate(self, rate):
 
     def write_bit_mode(self, mask, validate=False):
         """
-        handle; gained from device info
-        mask; value to write for the mask
+        :param mask: value to write for the mask
```

Comment:
I presume this was just to fix the warning due to the bad docstring format?

the resultant doc is indecipherable since it's the same thing as markdown where you need a blank line to actually force a newline/paragraph. I think perhaps the below will give the intended look.

```
:param mask: value to write for the mask for ``BIT_MODE.FT_BITMODE_CBUS_BITBANG``

* upper nibble is input (0) output (1)
* lower nibble is pin value low (0) high (1)
```

note list, and inline code segments use [double back-ticks](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup) in RST

Reply:
Yep, fixed.